src/elements/content-sidebar/activity-feed/activity-feed/__tests__/ActivityThread.test.js

```diff
@@ -1,22 +1,56 @@
 // @flow
-import React from 'react';
-import { shallow, ShallowWrapper } from 'enzyme';
 
+import React from 'react';
+import { render, screen, fireEvent } from '@testing-library/react';
 import ActivityThread from '../ActivityThread.js';
+import replies from '../../../../../__mocks__/replies';
 
 describe('src/elements/content-sidebar/activity-feed/activity-feed/ActivityThread', () => {
-    const getWrapper = (props = {}): ShallowWrapper => shallow(<ActivityThread {...props}>Test</ActivityThread>);
+    const defaultProps = {
+        replies,
+        total_reply_count: 2,
+        hasReplies: true,
+    };
+
+    const renderActivityThread = props =>
+        render(
+            <ActivityThread {...defaultProps} {...props}>
+                Test
+            </ActivityThread>,
+        );
+
+    test('should render children component wrapped in ActivityThread if hasReplies is true', () => {
+        const { container } = renderActivityThread();
+
+        expect(container.querySelector('.bcs-ActivityThread')).toBeInTheDocument();
+        expect(container).toHaveTextContent('Test');
```

Comment:
In the same way, instead of asserting which dom element has what text which results in more fragile test code, could we just assert that text is visible?
```
expect(screen.getByText('Test')).toBeVisible();
```